content/about/verslagen/17-18/2018-04-19.md

```diff
@@ -0,0 +1,174 @@
+% Bestuursvergadering 3
+% Isaura Claeys
+% 20/04/2018 - 16:00
+
+# Bestuursvergadering 5
+
+_Datum: 19/04/2018 - 16:00_
+_Status: Gesloten_
+
+**Timetable:**
+
+| Start | Duur | Onderwerp    |
+| ----- | ---- | ------------ |
+| 16:00 | '10  | Intro        |
+| 16:10 | '5   | Financiën    |
+| 16:15 | '5   | Sysadmin     |
+| 16:20 | '5   | Projecten    |
+| 16:25 | '20  | Events       |
+| 16:45 | '15  | Pauze        |
+| 17:00 | '20  | Ereleden     |
+| 17:20 | '20  | Verkiezingen |
+| 17:40 | '20  | Partnerships |
+| 18:00 | ...  | Trivia, ...  |
+
+
+## 0. Verslagen workflow
+Gebruik van hackmd.io: collaboratief verslagen schrijven in markdown. :white_check_mark: 
+ 
+- Goedkeuringen door middel van 'reviews' op PR's
+- Verslag op de site: officiëler en permanenter
+- Hopelijk een verbetering in de kwaliteit van verslagen tot gevolg
+- We vereisen dat elk bestuurslid het verslag goedkeurt op de PR
+
+## 1. Verslag vorige vergadering
+[Bestuursvergadering 4](https://drive.google.com/open?id=1l2LuBNQlMPtEKhjJ7ahJC3MRpc3l5fC9H2X7cvaCWQ4) :white_check_mark: 
+
+## 2. Financiën
+### Status
+€1100 + €200 ongeveer aan subsidies, wat vrij degelijk is voor Zeus.
```

Comment:
Waar vind ik eigenlijk gedetailleerde informatie over de financiele status en de veranderingen daarin? Is er een bezwaar tegen deze ook publiek beschikbaar (en goed vindbaar) te maken?

Reply:
Momenteel staan deze op de drive en zijn deze dus vrij afgeschermd, maar ik ben zeker en vast wel te vinden om deze informatie op de een of andere manier publiekelijk beschikbaar te maken, zeker een werkpunt naar volgend jaar toe!

Reply:
Zoals @Silox opmerkte weet ik niet in welke mate het goed idee is om echt alle geldstatus-informatie publiek te maken. We zouden eens goed moeten kijken wat we wel of niet willen transparant maken.

Maar op zich ben ik wel te vinden in een transparante boekhouding, maar dan eerder op een  'high-level' niveau.

Reply:
Over hoeveel geld er in de kassa zit caret niemand, wel hoeveel geld we hebben en waar dat naartoe gaat.